buildah.go

```diff
@@ -19,7 +19,7 @@ const (
 	// identify working containers.
 	Package = "buildah"
 	// Version for the Package
-	Version       = "0.0.1"
+	Version       = "0.1"
 	containerType = Package + " 0.0.1"
```

Comment:
It can be.  We use this to check if the data we've stored for a working container looks like it matches our version, and that data structure hasn't changed incompatibly since we moved to keeping both docker-style and OCI-style configuration structures separately.